var/spack/repos/builtin/packages/py-horton/package.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import platform
+
+
+class PyHorton(PythonPackage):
```

Comment:
I'm uncomfortable with calling this `py-horton`. There is another `horton` package on PyPI, and we usually default to PyPI naming schemes when naming Spack packages. Since this package isn't on PyPI, you can name it whatever you want as long as that name is not already in use. Perhaps `py-theochem-horton`.

Reply:
I've renamed it py-chem-horton.  The other horton doesn't seem to be very well known.